### PR TITLE
chore: removing SESSION_REPLAY_MOBILE_ONBOARDING flag

### PR DESCRIPTION
**Pull request by marandaneto**
## Problem

follow up https://github.com/PostHog/posthog/pull/25234
flag SESSION_REPLAY_MOBILE_ONBOARDING is active for months for everyone.
let's remove it and simplify the code a bit.

## Changes

👉 _Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review._

### CODE REVIEW

File: frontend/src/scenes/onboarding/sdks/session-replay/SessionReplaySDKInstructions.tsx [hunk 1]
```diff
@@ -15,6 +17,7 @@ import {
     VueInstructions,
     WebflowInstructions,
 } from '.'
+import { RNInstructions } from './react-native'
```
Comment (marandaneto) [inline frontend/src/scenes/onboarding/sdks/session-replay/SessionReplaySDKInstructions.tsx:20]: The linter is moving it here even if I import in the block above

File: frontend/src/scenes/onboarding/sdks/session-replay/SessionReplaySDKInstructions.tsx [hunk 2]
```diff
@@ -30,4 +33,7 @@ export const SessionReplaySDKInstructions: SDKInstructionsMap = {
     [SDKKey.SVELTE]: SvelteInstructions,
     [SDKKey.VUE_JS]: VueInstructions,
     [SDKKey.WEBFLOW]: WebflowInstructions,
+    [SDKKey.IOS]: iOSInstructions,
+    [SDKKey.ANDROID]: AndroidInstructions,
```
Comment (pauldambra) [inline frontend/src/scenes/onboarding/sdks/session-replay/SessionReplaySDKInstructions.tsx:37]: we're adding these two in `Onboarding.tsx`

```
    const sdkInstructionMap = SessionReplaySDKInstructions
    if (hasMobileOnBoarding) {
        sdkInstructionMap[SDKKey.ANDROID] = AndroidInstructions
        sdkInstructionMap[SDKKey.IOS] = iOSInstructions
    }
    ```
    
    which is controlled in there by
    
    ```
const { featureFlags } = useValues(featureFlagLogic)
    const hasMobileOnBoarding = !!featureFlags[FEATURE_FLAGS.SESSION_REPLAY_MOBILE_ONBOARDING]
```

I guess we can remove that flag and check here too?

Comment (pauldambra) [reply frontend/src/scenes/onboarding/sdks/session-replay/SessionReplaySDKInstructions.tsx:37]: yep, been out for 100% of users for months

https://us.posthog.com/project/2/feature_flags/45369

Comment (marandaneto) [reply frontend/src/scenes/onboarding/sdks/session-replay/SessionReplaySDKInstructions.tsx:37]: true that, good point, pushed a PR

Comment (marandaneto) [reply frontend/src/scenes/onboarding/sdks/session-replay/SessionReplaySDKInstructions.tsx:37]: mind reviewing https://github.com/PostHog/posthog/pull/25237 as well? both will probably conflict

**Review by pauldambra — APPROVED** [review]
ah. it's removed in this one :)

**Pull request merged**